frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -294,6 +294,19 @@ function ReaderHighlight:genPanelZoomMenu()
             end,
             separator = true,
         },
+        {
+            text = _("Fallback text selection"),
+            checked_func = function()
+                return self.fallback_text_selection
+            end,
+            callback = function()
+                self:onToggleFallbackTextSelection()
+            end,
+            hold_callback = function()
+                G_reader_settings:saveSetting("fallback_text_selection", self.fallback_text_selection)
```

Comment:
But the setting should have the `panel_zoom_` prefix, so we know what it's about/its context when viewing settings.reader.lua.